docs/pushpayloads.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "title":"FxA Push Payload schema",
+  "description":"This schema defines what is acceptable to send as a payload data with the Push API from the FxA servers to a device connected to FxA",
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "type":"object",
+  "required":[
+    "version",
+    "data"
+  ],
+  "properties":{
+    "version":{
+      "type":"number",
+      "description":"The version of this push payload data instance. Bump this if you make changes to any part of this schema."
```

Comment:
I'm totally making this up as I go, but given that part of the justification for "version" is to ensure we continue to support old clients correctly, I'd expect we need to think about this some more.

Eg, let's say we later found we wanted the device ID in the payload - do we simply bump the version and add the new field, or will we need *both* versions in the schema?  (I'm not sure how we would leverage this in practice though without some knowledge of what version the client at the other end actually expects)

So I guess I was expecting a very tight schema which reflects:

If action == "newDevice", { version, data } must be one of {1, "$ref": "#/definitions/newDevice/1} or {2, "$ref": "#/definitions/newDevice/2} etc

which would appear to make the version number less about the schema itself and more about the sub-schemas (is that a thing) - ie, each different action could potentially have multiple version numbers defined, each pointing at an immutable and very tight schema definition (and by tight, things like addtionalProperties: false would be specified).

(And in this specific case, I think we should include the deviceID in the payload)

What do ppl think?